src/planscape/conditions/load.py

```diff
@@ -21,40 +20,51 @@
     load.load_metrics('sierra_cascade_inyo')
 """
 
-def convert_metrics_nodata(region: str):
+
+def convert_metrics_nodata(region_name: str):
     """Replaces the NoData pixels in metric rasters with the standard NaN value and updates the raster profile accordingly.
 
     Args:
       region: The region to rewrite
     """
-    config_path = os.path.join(
-        PLANSCAPE_ROOT_DIRECTORY, 'src/planscape/config/conditions.json')
+    config_path = os.path.join(settings.BASE_DIR, 'config/conditions.json')
     config = PillarConfig(config_path)
+    data_path = os.path.join(settings.BASE_DIR, '../..')
 
-    region = config.get_region(region)
+    region = config.get_region(region_name)
+    if region is None:
+        return
     for pillar in config.get_pillars(region):
-        if not pillar['display']:
+        if not pillar.get('display', False):
             continue
         for element in config.get_elements(pillar):
             for metric in config.get_metrics(element):
                 # TODO: Update to interprted when available
                 for metric_type in ['.tif', '_normalized.tif']:
-                    metric_is_raw = True if metric_type == '.tif' else False
+                    metric_is_raw = (metric_type == '.tif')
 
                     reader = ConditionReader()
                     nan_condition = convert_metric_nodata_to_nan(
                         reader, metric, ConditionScoreType.CURRENT, metric_is_raw)
+                    if nan_condition is None:
+                        continue
 
                     # Overwrite original local version
```

Comment:
Could this check be made immediately after "for metric in config.get_metrics(element):"?

Reply:
oops - I meant to highlight
`metric_filepath = metric.get('filepath', None)
                    if metric_filepath is None:
                        continue` and ask whether that can be moved immediately after `for metric in config.get_metrics(element):`
                        
(but it also seems like nit now that I've read through the code!)

Reply:
Good idea.  Done.